src/compas_view2/views/view.py

```diff
@@ -284,6 +284,10 @@ def keyPressEvent(self, event):
         if key == QtCore.Qt.Key_Control:
             self.app.selector.mode = "deselect"
             self.keys["control"] = True
+        if key == QtCore.Qt.Key_F and self.app.selector.selected:
+            self.keys["F"] = True
```

Comment:
why is F here Upper case but in init it is small case, double check?

Reply:
Done, please review